lib/workers/repository/dependency-dashboard.ts

```diff
@@ -333,14 +334,14 @@ export async function ensureDependencyDashboard(
       'This repository currently has no open or pending branches.\n\n';
   }
 
-  issueBody += PackageFiles.getDashboardMarkdown(config);
+  // fit the detected dependencies section
+  const footer = getFooter(config);
+  issueBody += PackageFiles.getDashboardMarkdown(
+    config,
+    GitHubMaxPrBodyLen - issueBody.length - footer.length
```

Comment:
Can you add an issue for us to make this more generic across platforms? Eg maybe platforms can export maximumMarkdownLength or something like that so that this can work on gitlab too

Reply:
- https://github.com/renovatebot/renovate/issues/17174